### DISCUSSION TITLE
blog/dynamics-365-finance-importing-financial-balances-complete-guide

### DISCUSSION THREAD

**Discussion by giscus** [mrlopezco/lopezconextjs#5] [General]
# blog/dynamics-365-finance-importing-financial-balances-complete-guide

Learn how to import financial balances for the most important modules into Dynamics 365 with this blog post!

https://www.lopez-coll.com/blog/dynamics-365-finance-importing-financial-balances-complete-guide

**Comment by srizwan19** [comment]
where are the attachments in this blog ?

**Reply by mrlopezco** [reply]
Upsi dupsi! Attachments are now uploaded for this article. Just recently moved to a new website. Thanks for your patience!

**Comment by Peswarreddy1** [comment]
Thanks excellent explanation about the import opening 

Where are the attchements 

if those moved to new website pls provide website


it will help us

**Reply by mrlopezco** [reply]
Hi @Peswarreddy1  , you can find the attachments on the left hand sidebar. Cheers!
